service/telemetry/config.go

```diff
@@ -103,3 +104,12 @@ type MetricsConfig struct {
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
 }
+
+// TracesConfig defines settings to configure context propagation of the collector traces. Collector uses
+// go.opentelemetry.io/contrib/propagators/autoprop package to propagate the context and OTEL_PROPAGATORS
+// environment variable can be set to configure the propagators.
+type TracesConfig struct {
+	// DisablePropagation disables propagation of the trace context to external backend beyond exporters.
+	// The context is propagated by default.
```

Comment:
We should have it off by default for now, and think about having it on by default later, after it has had some testing.

Reply:
@jpkrohling let me know how to proceed. Per your comment in https://github.com/open-telemetry/opentelemetry-collector/issues/5572 I have enabled it by default, shall I heed @mx-psi advice here?

Reply:
In the long term, it should be enabled by default, so that we won't break the chain. Given that this is a change in the current behavior, this should be a two-step change: implement the new feature and make it disabled by default, but leave a note to the changelog stating that this will be the enabled by default by N+2.

Reply:
Thanks @jpkrohling - I have set the propagation to false by default for now - @mx-psi please review and let me know.

Reply:
Should this even have an enable/disable flag in the config, or should it be governed by a feature gate until we're comfortable enabling it by default?  The user can always use the `none` "propagator" to disable context propagation.

Reply:
per suggestion from @Aneurysm9 - the propagation is enabled by default behind a feature flag :)